x/finality/types/expected_keepers.go

```diff
@@ -17,6 +17,7 @@ type BTCStakingKeeper interface {
 	GetBTCStakingActivatedHeight(ctx context.Context) (uint64, error)
 	GetVotingPowerDistCache(ctx context.Context, height uint64) (*bstypes.VotingPowerDistCache, error)
 	RemoveVotingPowerDistCache(ctx context.Context, height uint64)
+	GetEpoch(ctx context.Context) uint64
```

Comment:
I would say that instead adding this function to `BTCStakingKeeper` , finality provider keep should now have dependency to `EpochingKeeper` as it is epoching keeper job to inform other modules about epochs

Reply:
Good point. I think this also makes sense to move `GetLastFinalizedEpoch` to `CheckpointingKeeper`